CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+## CHANGELOG
+
+***This document logs all changes to the core code or individual modules which cause a 
+variation in the behaviour of the software for users. This log will be used to write 
+out the final release notes at the moment of tagging a release.***
+
+- When adding content to this document, make sure to create a section for each module 
+if the changes only impact a single module and that section does not already exist in
+the document. When changes affect the entire software, make sure to add them in the 
+core section.
+
+- When possible please provide the number of the pull request(s) containing the 
+changes in the following format: PR #1234
+
+
```

Comment:
There should be a version number header here, since it's a changelog it will track more than 1 release.

Reply:
the problem with that is people submitting content to the version number but their PRs only being merged for another version... it's the same issue with preselecting milestones and stuff

i think we should put the version number before release

Reply:
They add it to the top (whatever the next version is going to be) and then we change the heading to the version number before the release